--- origsrc/postgresql-9.3.2/src/Makefile.shlib	2013-12-02 14:57:48.000000000 -0600
+++ src/postgresql-9.3.2/src/Makefile.shlib	2014-01-01 03:06:49.534835400 -0600
@@ -378,7 +378,7 @@ exports_file = $(DLL_DEFFILE)
 $(exports_file): $(OBJS)
 	$(DLLTOOL) --export-all $(DLLTOOL_DEFFLAGS) --output-def $@ $^
 else
-DLL_DEFFILE = lib$(NAME)dll.def
+DLL_DEFFILE = $(srcdir)/lib$(NAME)dll.def
 endif
 
 $(shlib): $(OBJS) $(DLL_DEFFILE) | $(SHLIB_PREREQS)
