--- origsrc/postgresql-9.5.1/src/Makefile.shlib	2016-02-08 15:12:28.000000000 -0600
+++ src/postgresql-9.5.1/src/Makefile.shlib	2016-03-04 15:35:44.787300900 -0600
@@ -427,7 +427,7 @@ UC_NAME = $(shell echo $(NAME) | tr 'abc
 
 lib$(NAME)dll.def: $(SHLIB_EXPORTS)
 	echo '; DEF file for win32.mak release build and for Makefile.shlib (MinGW)' >$@
-	echo 'LIBRARY LIB$(UC_NAME).dll' >>$@
+	echo 'LIBRARY lib$(NAME).dll' >>$@
 	echo 'EXPORTS' >>$@
 	sed -e '/^#/d' -e 's/^\(.*[ 	]\)\([0-9][0-9]*\)/    \1@ \2/' $< >>$@
 
